src/pages/dmt5/_flexibility.tsx

```diff
@@ -125,7 +133,7 @@ const StyledText = styled(Text)`
 `
 
 const Flexibility = () => {
-    const { is_eu_country } = React.useContext(DerivStore)
+    const is_eu_country = React.useContext(DerivStore)
```

Comment:
Any reason we are removing this?

Reply:
Any reason we are removing this?
> 
<img width="1792" alt="Screenshot 2021-12-29 at 15 52 20" src="https://user-images.githubusercontent.com/94372063/147664421-aff60e79-1c82-4fd0-957b-820fd37eb966.png">

Yes, think that ts warned me. Do I need to return {} ?

Reply:
Added {is_eu_country}